config/routes.rb

```diff
@@ -1,8 +1,9 @@
 Rails.application.routes.draw do
 
   resources :github_events, :only => [ :index, :show ]
+  resources :repos, :only => [ :index, :show ]
 
-  get '/auth/:provider/callback' => 'sessions#create'
+  get '/auth/:provider/callback' => 'sessions#create', :as => :auth_create
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.